diff --git a/node_modules/@greenwood/plugin-renderer-lit/src/index.js b/node_modules/@greenwood/plugin-renderer-lit/src/index.js
index c5979fa..2c98c59 100755
--- a/node_modules/@greenwood/plugin-renderer-lit/src/index.js
+++ b/node_modules/@greenwood/plugin-renderer-lit/src/index.js
@@ -6,9 +6,9 @@ class LitHydrationResource {
 
   async shouldIntercept(url) {
     const { pathname } = url;
-    const matchingRoute = this.compilation.graph.find((node) => node.route === pathname) || {};
+    const matchingRoute = this.compilation.graph.find((node) => node.route === pathname);
 
-    return matchingRoute.isSSR && matchingRoute.hydration;
+    return matchingRoute && ((matchingRoute.isSSR && matchingRoute.hydration) || this.compilation.config.prerender);
   }
 
   async intercept(url, request, response) {
